app/models/miq_user_role.rb

```diff
@@ -112,6 +112,10 @@ def tenant_admin_user?
     allows?(:identifier => MiqProductFeature::TENANT_ADMIN_FEATURE)
   end
 
+  def report_only_my_user_tasks?
```

Comment:
Following the convention of the above methods, how about naming this one `my_tasks_user?`

Reply:
@gtanzillo Thank you for the feedback.

I named this method `report_only_my_user_tasks?`  because it is possible that `miq_task_my_ui` product feature is enabled `and` `miq_task_all_ui` in which case naming this method `my_tasks_user?` would be somewhat inaccurate. One might expect `my_tasks_user?` to return true if `miq_task_my_ui` is set but that is not the functionality we want. We want a method to return true when only `miq_task_my_ui` is set and not `miq_task_all_ui`.

Would you be OK if I renamed this method from `report_only_my_user_tasks?` to `only_my_user_tasks?` ?